ideascube/views.py

```diff
@@ -25,23 +25,49 @@
 from ideascube.library.models import Book
 from ideascube.mediacenter.models import Document
 
+# Do not import directly Catalog as we want to mock it for test.
+from ideascube.serveradmin import catalog as catalog_mod
+
 from .forms import UserForm, CreateStaffForm
 from .mixins import CSVExportMixin
 
 user_model = get_user_model()
 
+def build_package_card_info():
+    package_card_info = []
+    catalog = catalog_mod.Catalog()
+    packages_to_display = catalog.list_installed(get_config('ideascube', 'displayed-package-ids'))
```

Comment:
Things like these make me think that the `Catalog` API is not that bad after all. :smile: 

Line is too long, though. :wink: